go/vt/vtgate/evalengine/abs.go

```diff
@@ -0,0 +1,52 @@
+package evalengine
+
+import (
+	"math"
+	"vitess.io/vitess/go/sqltypes"
+	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
+	"vitess.io/vitess/go/vt/vterrors"
+)
+
+type buildinABS struct{}
```

Comment:
Built-in is spelled with a `t` :sweat_smile: -- also, the name of the function doesn't need to be in all caps. `builtinAbs` would be a better name.

Reply:
Okay! I will revise it.

Reply:
This question has been revised and submitted. Do you have any questions？